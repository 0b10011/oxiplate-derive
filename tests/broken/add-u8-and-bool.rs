use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline = "{{ a + b }}"]
struct Data {
    a: u8,
    b: bool,
}

fn main() {
    let data = Data { a: 4, b: false };

    print!("{}", data);
}
