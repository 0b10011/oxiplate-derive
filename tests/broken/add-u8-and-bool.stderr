error[E0277]: cannot add `bool` to `u8`
 --> tests/broken/add-u8-and-bool.rs:4:27
  |
4 | #[oxiplate_inline = "{{ a + b }}"]
  |                           ^ no implementation for `u8 + bool`
  |
  = help: the trait `Add<bool>` is not implemented for `u8`
  = help: the following other types implement trait `Add<Rhs>`:
            `&u8` implements `Add<u8>`
            `&u8` implements `Add`
            `u8` implements `Add<&u8>`
            `u8` implements `Add`
